lib/polyfill/Ripple.js

```diff
@@ -67,8 +67,8 @@ export default class Ripple extends Component {
                 activeOpacity={1}
                 onPress={onPress}
                 onLongPress={onLongPress}
-                onPressIn={this._highlight}
-                onPressOut={this._unHighlight}
+                onPressIn={() => this._highlight()}
+                onPressOut={() => this._unHighlight()}
```

Comment:
`this._highlight` and `this._unHighlight` are already defined as long arrow functions, so I'm not sure what this is fixing? ... scope shouldn't be lost, and the methods should be getting called correctly?